Cubical/Tactics/FunctorSolver/Solver.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --safe --lossy-unification #-}
```

Comment:
Credit the 1lab?

Reply:
The implementations of the solvers are not taken from the 1lab, just the core reflection stuff in `Cubical.Tactics.Reflection`